sdk/python/symbolchain/sc/__init__.py

```diff
@@ -14976,7 +14976,7 @@ def deserialize(cls, payload: bytes) -> Block:
 			(NormalBlockV1.BLOCK_TYPE): NormalBlockV1,
 			(ImportanceBlockV1.BLOCK_TYPE): ImportanceBlockV1
 		}
-		discriminator = (parent.type_)
+		discriminator = parent.type_
```

Comment:
this is generated code, so need to update generator too ;)

Reply:
I just disable the rule since it is generated code.  Let me know if that's ok.